Use the Khronos include alias on every Apple target, including iOS Simulator.

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,7 +23,7 @@
 add_library(Headers INTERFACE)
 add_library(OpenCL::Headers ALIAS Headers)
 
-if(CMAKE_SYSTEM_NAME MATCHES Darwin)
+if(APPLE)
   execute_process(
     COMMAND ln -snf ${CMAKE_CURRENT_SOURCE_DIR}/CL ${CMAKE_CURRENT_BINARY_DIR}/CL
     COMMAND ln -snf ${CMAKE_CURRENT_SOURCE_DIR}/CL ${CMAKE_CURRENT_BINARY_DIR}/OpenCL
